explore/src/main/scala/explore/itc/package.scala

```diff
@@ -22,3 +22,10 @@ def requiredForITC: TagMod =
       <.span(^.cls := "fa-layers-text fa-inverse", "ITC")
     )
   )("Required for ITC")
+
+opaque type PlotLoading = Boolean
+
+object PlotLoading:
+  def apply(b: Boolean): PlotLoading = b
+
+extension (p: PlotLoading) def boolValue: Boolean = p
```

Comment:
We just merged a PR that introudes a `trait NewType[A]` that implements this opaque type boilerplate.